### DISCUSSION TITLE
执行./go-cqhttp时报错

### DISCUSSION THREAD

**Discussion by stars-spark** [koishijs/koishi#939] [Q&A]
### Describe the bug

如图
![IMG_20230118_104059](https://user-images.githubusercontent.com/53867496/213069711-d747fd71-f7bf-4701-9977-e95f13c91a86.jpg)

### Steps to reproduce

如题

### Expected behavior

正常运行

### Versions

- OS: 
- Backend service: 
- Node version: 
- Koishi version: 
最新版

**Comment by Seidko** [comment]
请在配置文件中修改监听端口

**Comment by stars-spark** [comment]
> 

有哪些可用的监听端口？

**Comment by stars-spark** [comment]
> 请在配置文件中修改监听端口

8080已被另一个机器人占用

**Comment by Seidko** [comment]
任意都可以，你需要在gocq的配置和koishi的配置都中修改成对应端口

**Comment by stars-spark** [comment]
> 请在配置文件中修改监听端口

已启用8443接口

**Comment by Seidko** [comment]
问题解决了吗？

另外，请尽量使用截图工具而不是拍屏

**Comment by stars-spark** [comment]
> 问题解决了吗？
> 
> 另外，请尽量使用截图工具而不是拍屏

谢谢已解决，下次尽量复制报错代码
